src/pages/interviews/devjourney.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: "Swizec Teller wanted to work on silicon valley blockbusters"
+description: "On the DevJourney podcast Swiz' told us how he discovered computers at a very early age and how the planets aligned. By the time he enrolled in a CS degree in Slovenia, he had already had multiple internships and experiences working as a software developer. He spoke about freelancing and how this led him not to finish his CS degree. He talked about how his first entrepreneurial experience took him to Silicon Valley, what he learned there (by crashing and burning his startup) and how it eventually took him where he worked for the past seven years."
+hero: "../../images/swizec-interview-face.jpg"
+published: 2022-05-17
+---
+
+On the DevJourney podcast Swiz' told us how he discovered computers at a very early age and how the planets aligned. By the time he enrolled in a CS degree in Slovenia, he had already had multiple internships and experiences working as a software developer. He spoke about freelancing and how this led him not to finish his CS degree. He talked about how his first entrepreneurial experience took him to Silicon Valley, what he learned there (by crashing and burning his startup) and how it eventually took him where he worked for the past seven years.
+
+<iframe style="border-radius:12px" src="https://open.spotify.com/embed/episode/36cOMQZCkjsTE8baZt82VR?utm_source=generator&t=0" width="100%" height="232" frameBorder="0" allowfullscreen="" allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture"></iframe>
+
+[DevJourney Podcast Episode #202](https://devjourney.info/Guests/202-SwizecTeller.html)
+
+## Talking points
+
+- Swizec's journey into programming
+- Running out of variables in Pascal
+- Swizec got in trouble with Paul Graham and Ycombinator and nearly got his startup kicked out.
+- the biggest mindset shifts that helped me go from code monkey to tech lead* Moving to Silicon Valey
+- Contributing code to a nuclear power plant
+- Working for free on open-source vs working for money
+- The sad story of CURL, the command line utility
+- Innovation and entrepreneurship
+- The biggest mistake we made in startup fundraising
+- How Swizec did an intersnhip in Palo Alto and ended up living next door to Paul Graham.
+- You should be in the Silicon Valley ecosystem if you want to make it really big
+
+## Quotable Quotes
+
+"you know how some people, some kids get really obsessed with, uh, action figures and action heroes and stuff. For me, that was programming" - ST
+
+"I was in my twenties and I didn't really realize that programming concepts actually had meanings in reality. LIN means 'write line' because you're writing out a lin.I remember waking up and like, oh my God, these things have actual meanings. That's amazing. It's makes it so much easier." - ST
```

Comment:
```suggestion
"I was in my twenties and I didn't really realize that programming concepts actually had meanings in reality. `WriteLn` means 'write line' because you're writing out a line. I remember waking up and like, oh my God, these things have actual meanings. That's amazing. It's makes it so much easier." - ST
```